Dice.js

```diff
@@ -0,0 +1,9 @@
+const rollDice = function (num = 6){
+    var num1 = Math.floor(Math.random() * Math.floor(num));
```

Comment:
your random generator gives you a random number from 0 - 6. 

do the following to give you a random number between 1 - 6
`Math.floor(Math.random()*6)+1;`